src/java.desktop/share/classes/javax/swing/SwingWorker.java

```diff
@@ -304,13 +304,14 @@ public T call() throws Exception {
                         try {
                             return doInBackground();
                         } finally {
-                            setState(StateValue.DONE);
                             doneEDT();
+                            setState(StateValue.DONE);
                         }
                     }
                 };
 
-       future = new FutureTask<T>(callable);
+        future = new FutureTask<T>(callable);
```

Comment:
```suggestion
       future = new FutureTask<T>(callable);
```
An extra space breaks the correct indentation.